src/actions/editUserActions.js

```diff
@@ -1,58 +1,54 @@
-let { writeFile, readFile, parseJSON } = require("../helpers/helpers")
+let User = require("../models/userModel");
 
-let dataPATCH = (req, res, data) => {
-
-    const body = req["body"];
-    const id = Number(req["params"].id)
+const editUserPut = (req, res) => {
+    if(Object.keys(req.body).length===4){
+    User.findByIdAndUpdate(req.params.id, { $set: req.body }, { new: true }).then(user => {
+        reqValidData(req.body, user, 1)
```

Comment:
can you chain promises so you don't have a `.then()` inside another `.then()`

Reply:
We placed this in our TODO list.